javascripts/discourse/components/discourse-tag-banners-presentation.js

```diff
@@ -0,0 +1,5 @@
+import Component from "@ember/component";
+
+export default class DiscourseTagBannersPresentation extends Component {
+  tagName = "";
+}
```

Comment:
I believe these empty files aren't necessary... unless you're doing this so the class name for the component can be `tag-banner__text-only` exactly?